src/clj/game/cards/hardware.clj

```diff
@@ -414,6 +414,24 @@
                                {:effect (effect (add-counter :runner card :virus 1)
                                                 (system-msg :runner (str "places 1 virus counter on Friday Chip")))}}}}})
 
+   "Gebrselassie"
+   {:abilities [{:msg (msg "host it on an installed non-AI icebreaker")
+                 :cost [:click 1] 
+                 :choices {:req #(and 
+                                   (installed? %) 
+                                   (has-subtype? % "Icebreaker") (not (has-subtype? % "AI")))}
```

Comment:
My preferred style here is for the first function to go on the same line as the `and`, then the other two to be on their own lines.